letsdrill.rb

```diff
@@ -1,17 +1,25 @@
 def get_letter_grade(integer)
- 
-  #Put your code here!
-
+  if integer < 60
+    return "F"
+  elsif integer >= 60 && integer < 70
+  	return "D"
+  elsif integer >= 70 && integer < 80
+  	return "C"
+  elsif integer >= 80 && integer < 90
+  	return "B"
+  else 
+  	return "A"
+  end
 end
 
 def shortest_string(array)
-
-  #Put your code here!
-
+  string_length = array.sort_by { |word| word.length }
+  string_length.reverse
```

Comment:
Nice approach on this problem.  Is "string_length" really the most apt variable name here?